CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+# 0.41.0
+
+- Packit now supports `changelog-entry` action that is used when creating
+  SRPM, the action is supposed to generate whole changelog entry (including
+  the `-` at the start of the lines) and has a priority over any other way we
+  modify the changelog with. (#1367)
+- Fixed an issue, which raised a `UnicodeEncodingError`, when working with
+  dist-git patch files with an encoding other than UTF-8. (#1406)
+- Backup alias definitions now reflect the official release of Fedora Linux 35. (#1405)
+- We have introduced a new configuration option `merge_pr_in_ci` that allows
+  you to disable merging of PR into the base branch before creating SRPM in
+  service. (#1395)
+- Fixed an issue, where spec-files located in a sub-directory of upstream
+  projects, were not placed in the root of the dist-git repo when proposing
+  changes downstream. (#1402)
+
```

Comment:
I would also mention #1397 (I am not sure if this one should be mentioned in previous release but I can't see it there)

Reply:
Nice catch! I see it was merged around the time the release notes for 0.40.0 were prepared, so maybe that's the reason it was missed. I'll add it to the changelog for the previous release and update the release notes on GitHub with it.